Lib/test/test_os.py

```diff
@@ -2206,8 +2206,11 @@ def test_listdir(self):
         # test listdir without arguments
         current_directory = os.getcwd()
         try:
-            os.chdir(os.sep)
-            self.assertEqual(set(os.listdir()), set(os.listdir(os.sep)))
+            with tempfile.TemporaryDirectory() as tmpdir:
+                os.chdir(tmpdir)
+                open("a.txt", "w").close()
+                open("test_file.foo", "w").close()
+                self.assertEqual(set(os.listdir()), set(os.listdir(tmpdir)))
```

Comment:
There is already a directory with a known content: `self.dir`.
```suggestion
            os.chdir(self.dir)
            self.assertEqual(set(os.listdir()), expected)
```